README.md

```diff
@@ -83,7 +95,7 @@ it is faster to update the tree, instead of rebuilding it from scratch.
 First of all, optimizing is not helpful if more than half of the scene is not static.
 This is due to how optimizing takes place:
 Given a set of indices of all shapes which have changed, the optimize procedure tries to rotate fixed constellations
-in search for a better surface area heuristic (SAH) value. This is done recursively from bottom to top while fixing the AABBs
+in search for a better surface area heuristic (SAH) value. This is done recursively from bottom to top while fixing the Aabbs
```

Comment:
There are still a few miss-capitalized occurrences like this in prose and docs. Could you do another pass?

Reply:
Woops, fixed! I didn't actually find any in the docs though.